src/setupTests.ts

```diff
@@ -1,10 +1,10 @@
 import '@testing-library/jest-dom' // jest custom assertions
 import 'jest-styled-components' // adds style diffs to snapshot tests
 
+import { ResizeObserver } from '@juggle/resize-observer'
```

Comment:
Instead of this, can you just `import 'polyfills'` on L3?

Reply:
gonna do in separate PR so I don't need another approval on this before merging